docs/Rakefile

```diff
@@ -98,15 +99,18 @@ end
 
 desc 'Create adoc variables'
 task :adoc_variables => :init do
-  minishift_version  = ENV['MINISHIFT_VERSION'] || 'unset'
-  openshift_version  = ENV['OPENSHIFT_VERSION'] || 'unset'
-  centos_iso_version = ENV['CENTOS_ISO_VERSION'] || 'unset'
+  minishift_version    = ENV['MINISHIFT_VERSION']    || 'unset'
+  openshift_version    = ENV['OPENSHIFT_VERSION']    || 'unset'
+  centos_iso_version   = ENV['CENTOS_ISO_VERSION']   || 'unset'
+  minikube_iso_version = ENV['MINIKUBE_ISO_VERSION'] || 'unset'
```

Comment:
You are doing this for completeness reasons, right? It is not uses anywhere yet?

Reply:
The environment variable is already being passed along to `rake` during the build -- I assumed that adding it to the Rakefile was overlooked, and that it was going to be used in the future.

Reply:
+1